src/components/actionBar/actionBar/ActionBar.tsx

```diff
@@ -0,0 +1,176 @@
+import {
+    Stack,
+    Autocomplete,
+    TextField,
+    InputAdornment,
+    IconButton,
+} from '@mui/material';
+import {
+    PanoramaFishEyeSharp,
+    FmdGood,
+    Today,
+    People,
+    Search,
+    SwapHoriz,
+} from '@mui/icons-material';
+import { DatePicker } from '@mui/x-date-pickers';
+import React, { useState } from 'react';
+import FilterSort from '../filterSort/FilterSort';
+import { CenteredButton, Wrapper } from './PnrSearch.styled';
+
+interface IActionBarProps {
+    showFilterSort?: boolean;
+}
+
+// interface in the mui format
+interface ILocationOptions {
+    id: number;
+    label: string;
+}
```

Comment:
Move interfaces into types file.

Reply:
Moving common interface to types file, retaining locally used interface IActionBarProps within the file.